sbnanaobj/StandardRecord/SRBlip.h

```diff
@@ -0,0 +1,44 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRBlip.h
+// \brief   SRBlip object for localized energy deposits in bulk LAr
+// \author  $Author: jmclaughlin2@illinoistech.edu
+////////////////////////////////////////////////////////////////////////
+#ifndef SRBLIP_H
+#define SRBLIP_H
+#include "sbnanaobj/StandardRecord/SRVector3D.h"
+#include "sbnanaobj/StandardRecord/SRBlipHitClust.h"
+#include "sbnanaobj/StandardRecord/SRBlipTrueBlip.h"
+#include "sbnanaobj/StandardRecord/SRCVNScore.h"
+
+namespace caf
+{
+  const int kNplanes    = 3;
+    class SRBlip
```

Comment:
Add a description of the meaning of the class as Doxygen documentation.
You can start with something like:

```suggestion
    /**
     * @brief Small reconstructed isolated energy deposit ("blip").
     * 
     * A blip is informally defined as a small cluster of TPC hits with unclear relation
     * to other reconstructed objects.
     * ...
     */
    class SRBlip
```